contrib/docs-code-samples/expand-api-display-access/99-cleanup/index.js

```diff
@@ -1,39 +1,18 @@
 import grpc from "@ory/keto-grpc-client/node_modules/@grpc/grpc-js/build/src/index.js"
-import { write, writeService, read, readService } from "@ory/keto-grpc-client"
-
-const readClient = new readService.ReadServiceClient(
-  "127.0.0.1:4466",
-  grpc.credentials.createInsecure(),
-)
+import { relationTuples, write, writeService } from "@ory/keto-grpc-client"
```

Comment:
These read like a lot of changes. Are they due to a new API, or did you just refactor the test? How should users migrate to the new API?